schema.graphql

```diff
@@ -1,10 +1,12 @@
 type Fid @entity {
   id: BigInt!
   address: String!
+  fnames: [Fname] @derivedFrom(field: "fid")
```

Comment:
Common GraphQL footgun - the list type and element type should (almost always) both be non-nullable. Though I might be misunderstanding the relationship logic here. https://stackoverflow.com/a/46771890


```gql
fnames: [Fname!]! @derivedFrom(field: "fid")
```